.github/workflows/docs.yml

```diff
@@ -0,0 +1,47 @@
+name: UCX-Py documentation build
+
+on:
+  push:
+    branches: ["branch-0.20"]
+
+jobs:
+  build:
+    name: Build Sphinx documentation
+    runs-on: "ubuntu-latest"
+    defaults:
+      run:
+        shell: bash -l {0}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2.3.4
+      - name: Install libnuma
+        run: |
+          sudo apt-get update -y
+          sudo apt-get install -y libnuma-dev
+      - name: Create docs environment
+        uses: conda-incubator/setup-miniconda@v2
+        with:
+          activate-environment: ucx_dev
+          environment-file: conda/environments/builddocs_py37.yml
+          auto-activate-base: false
+      - name: Build documentation
+        run: |
+          conda activate ucx_dev
+          cd docs
+          make html
+      - name: Commit changes to docs
+        run: |
+          git clone https://github.com/rapidsai/ucx-py.git --branch gh-pages --single-branch gh-pages
```

Comment:
The build will fail here until we add a (preferably empty) `gh-pages` branch to this repo.

Reply:
I've just created an empty `gh-pages` branch for you. https://github.com/rapidsai/ucx-py/tree/gh-pages